ui/lib/pki/addon/components/pki-generate-root.hbs

```diff
@@ -37,11 +37,11 @@
       {{#if @model.privateKey}}
         <CertificateCard @data={{@model.privateKey}} />
       {{else}}
-        <span class="tag">internal</span>
+        <Hds::Badge @text="internal" />
       {{/if}}
     </InfoTableRow>
     <InfoTableRow @label="Private key type" @value={{@model.privateKeyType}}>
-      <span class="{{unless @model.privateKeyType 'tag'}}">{{or @model.privateKeyType "internal"}}</span>
+      <Hds::Badge @text={{or @model.privateKeyType "internal"}} />
```

Comment:
Similar to the previous file, this needs to be broken up depending on whether or not the value is returned 
```suggestion
     {{# if (@model.privateKeyType)}}
       {{@model.privateKeyType}}
     {{else}}
       <Hds::Badge @text=internal"/>
     {{/if}}
```